src/libraries/System.Text.RegularExpressions/tests/RegexParserTests.cs

```diff
@@ -875,45 +774,6 @@ private static void ParseSubTree(string pattern, RegexOptions options)
             }
         }
 
-        /// <summary>
-        /// Checks if action throws either a RegexParseException or an ArgumentException depending on the
-        /// environment and the supplied error.
-        /// </summary>
-        /// <param name="error">The expected parse error</param>
-        /// <param name="action">The action to invoke.</param>
-        private static void Throws(RegexParseError error, Action action)
-        {
-            // If no specific error is supplied, or we are running on .NET Framework where RegexParseException doesn't exist
-            // we expect an ArgumentException.
-            if (PlatformDetection.IsNetFramework)
-            {
-                Assert.ThrowsAny<ArgumentException>(action);
-                return;
-            }
-
-            try
-            {
-                action();
-            }
-            catch (ArgumentException e)
-            {
-                // We use reflection to check if the exception is an internal RegexParseException
-                // and extract its error property and compare with the given one.
-                if (e.GetType() == s_parseExceptionType)
-                {
-                    RegexParseError regexParseError = (RegexParseError)s_parseErrorField.GetValue(e);
-
-                    // Success if provided error matches.
-                    if (error == regexParseError)
-                        return;
-
-                    throw new XunitException($"Expected RegexParseException with error: ({error}) -> Actual error: {regexParseError})");
-                }
-
-                throw new XunitException($"Expected RegexParseException -> Actual: ({e.GetType()})");
-            }
-
-            throw new XunitException($"Expected RegexParseException with error: ({error}) -> Actual: No exception thrown");
-        }
+        static partial void Throws(RegexParseError error, int offset, Action action);
```

Comment:
Interesting, considering that we're doing partial classes, I would've thought we'd be able to delete this line. Just a nit. I don't consider it blocking.